### PR TITLE
Proposed CHANGES file for 1.22.1

### PR DESCRIPTION
**Pull request by bradcray**
This is an update to the CHANGES file in anticipation of a release of Chapel addressing gcc 10.x issues.

### CODE REVIEW

**Comment by bradcray** [comment]
@ronawho, @jhoole: Would you give me a review on this when you get a chance (e.g., next week).

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+
+version 1.22.1
+==============
+
+Update to twenty-fifth public release of Chapel, June 2020
+(see also changes below for 1.22.0)
+
+Portability
+-----------
+* addressed some coding issues that caused problems with gcc 10.x
```
Comment (ronawho) [inline CHANGES.md:13]: This reads odd to me. Maybe "Fixed portability issues with GCC 10"?

Comment (bradcray) [reply CHANGES.md:13]: To me, that makes it sound like gcc 10 itself has portability issues.  How about

```
* fixed portability issues in our code w.r.t. gcc 10.x
```

Comment (ronawho) [reply CHANGES.md:13]: works for me

**Pull request merged**